### PR TITLE
fix: add subtle link, improve the tests

### PR DESCRIPTION
**Pull request by droshev**
fixes: #786 
Screen Shot 2020-03-29 at 3 29 03 PM

### CODE REVIEW

File: src/mixins/_mixins.scss [hunk 1]
```diff
@@ -308,6 +308,10 @@
     color: var(--sapLink_InvertedColor);
   }
 
+  &--subtle {
```
Comment (salarenko) [inline src/mixins/_mixins.scss]: Shouldn't this `link` mixin be extracted to separate link component file?

**Comment by JKMarkowski** [comment]
Thsi variable doesn't have value, cause there is no `--sapLink_SubtleColor` in sap theming repo.

File: src/mixins/_mixins.scss [hunk 2]
```diff
@@ -308,6 +308,10 @@
...
+    color: var(--sapLink_SubtleColor);
```
Comment (InnaAtanasova) [inline src/mixins/_mixins.scss:312]: No such variable in @sap-theming

**Comment by droshev** [comment]
> This variable doesn't have value, because there is no `--sapLink_SubtleColor` in sap theming repo.

it has been released now

File: src/mixins/_mixins.scss (hunk 2 above)
Comment (droshev) [reply src/mixins/_mixins.scss:312]: it has been released

File: src/mixins/_mixins.scss (hunk 1 above)
Comment (droshev) [reply src/mixins/_mixins.scss:311]: @salarenko we have an inverted one here. How do you see the subtle to be in another file?

**Pull request merged**